coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -1098,13 +1098,16 @@ func (s *server) FailJob(ctx context.Context, failJob *proto.FailedJob) (*proto.
 func (s *server) notifyWorkspaceBuildFailed(ctx context.Context, workspace database.Workspace, build database.WorkspaceBuild) {
 	var reason string
 	if build.Reason.Valid() && build.Reason == database.BuildReasonInitiator {
-		return // failed workspace builds initiated by a user should not notify
+		return // failed workspace build initiated by a user should not notify
 	}
+	reason = "initiated by autobuild"
```

Comment:
Won't `BuildReasonAutostart` or `BuildReasonAutostop` be more correct here?
```suggestion
	reason = build.Reason.String()
```

Reply:
Fixed